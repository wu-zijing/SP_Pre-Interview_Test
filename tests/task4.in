4
2355
4663
2
9753