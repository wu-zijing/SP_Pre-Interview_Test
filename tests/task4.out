["bell", "cell"]
["gone", "good", "goof", "home", "hone", "hood", "hoof"]
["a", "b", "c"]
[""]